beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/Sync.java

```diff
@@ -32,6 +33,12 @@ public interface Sync {
 
   SafeFuture<Optional<SyncProgress>> getSyncProgress();
 
+  long subscribeToBlocksImportedEvent(BlocksImportedSubscriber subscriber);
+
+  interface BlocksImportedSubscriber {
+    void onBlocksImported(SignedBeaconBlock lastImportedBlock);
```

Comment:
why 's, plural?

Reply:
oh, problem is that I put that on `Sync` interface, and not `BatchSync` interface. So the concept of "batch" is not part of that...
Suggestions? :)